### PR TITLE
MGMT-1767 Move git repo ori-amizur/introspector to opnshift/assisted-installer-agent

### PR DESCRIPTION
**Pull request by ori-amizur**
Rename the docker image names to call single docker image (https://quay.io/repository/ocpmetal/assisted-installer-agent:latest) by default

### CODE REVIEW

File: internal/host/instructionmanager.go [hunk 1]
```diff
@@ -43,9 +43,9 @@ type InstructionConfig struct {
 	ServiceBaseURL         string `envconfig:"SERVICE_BASE_URL"`
 	InstallerImage         string `envconfig:"INSTALLER_IMAGE" default:"quay.io/ocpmetal/assisted-installer:latest"`
 	ControllerImage        string `envconfig:"CONTROLLER_IMAGE" default:"quay.io/ocpmetal/assisted-installer-controller:latest"`
-	ConnectivityCheckImage string `envconfig:"CONNECTIVITY_CHECK_IMAGE" default:"quay.io/ocpmetal/connectivity_check:latest"`
-	InventoryImage         string `envconfig:"INVENTORY_IMAGE" default:"quay.io/ocpmetal/inventory:latest"`
-	FreeAddressesImage     string `envconfig:"FREE_ADDRESSES_IMAGE" default:"quay.io/ocpmetal/free_addresses:latest"`
+	ConnectivityCheckImage string `envconfig:"CONNECTIVITY_CHECK_IMAGE" default:"quay.io/ocpmetal/assisted-installer-agent:latest"`
+	InventoryImage         string `envconfig:"INVENTORY_IMAGE" default:"quay.io/ocpmetal/assisted-installer-agent:latest"`
+	FreeAddressesImage     string `envconfig:"FREE_ADDRESSES_IMAGE" default:"quay.io/ocpmetal/assisted-installer-agent:latest"`
```
Comment (ronniel1) [inline internal/host/instructionmanager.go:48]: Maybe we can remove all these?

Comment (ori-amizur) [reply internal/host/instructionmanager.go:48]: Thought about it.  Do we want to have flexibility to test only one? Otherwise, we can unify them.

Comment (ronniel1) [reply internal/host/instructionmanager.go:48]: OK, we can leave it like this for now.

**Pull request merged**